ehrapy/data/_datasets.py

```diff
@@ -153,12 +161,13 @@ def diabetes_130(
 
     Examples:
         >>> import ehrapy as ep
-        >>> adata = ep.dt.diabetes_130(encoded=True)
+        >>> adata = ep.dt.diabetes_130_raw(encoded=True)
     """
+
     adata = read_csv(
-        dataset_path=f"{ehrapy_settings.datasetdir}/diabetes_130.csv",
-        download_dataset_name="diabetes_130.csv",
-        backup_url="https://figshare.com/ndownloader/files/33950546",
+        dataset_path=f"{ehrapy_settings.datasetdir}/diabetes_130_raw.csv",
+        download_dataset_name="diabetes_130_raw.csv",
+        backup_url="https://figshare.com/ndownloader/files/33950546",  # TODO: change link, or download zip and extract?
```

Comment:
This is where the question from the first comment applies

Reply:
Starting to think figshare is indeed a neat and quick way instead of dwnloading things from there and extracting